src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -98,12 +99,20 @@ private IVsStatusbar VsStatusBar
         {
```

Comment:
It seems VSUIShell property on line 92 has a replaceable GetService() call.

Reply:
This one is called only 1 place [Line#680](https://github.com/NuGet/NuGet.Client/blob/cb952a060098aa561d998dfcc1f18e78e1b72367/src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs#L680) which is synchronous call. 
So if we make this one async then we have to add JTF in Line#680 then it cancels all the benefit and we have to pay for cost of state machine.